internal/olm/operator/bundle/install.go

```diff
@@ -78,6 +78,12 @@ func (i *Install) setup(ctx context.Context) error {
 	i.OperatorInstaller.PackageName = labels["operators.operatorframework.io.bundle.package.v1"]
 	i.OperatorInstaller.CatalogSourceName = fmt.Sprintf("%s-catalog", i.OperatorInstaller.PackageName)
```

Comment:
IMO the `setup` would not be required if we have a newOperatorInstaller(values) and this complexity was abstracted in its entity class definition (DDD principle). However, shows that the goal of your PR is just appending +1 value and it would be out of this scope. Then, OK 👍

Reply:
@camilamacedo86 I agree with you. I wanted to do that in this PR but it was taking so long to get the basic logic done I didn't want to add refactoring the NewOperatorInstaller method. In my notes on the code I wrote that NewOperatorInstaller should take in everything it needs to create a populated OperatorInstaller.  This required more changes so I will do that in a follow on. Sorry to add more tech debt. You can see my notes in the image below.
![photo_2020-09-17_11-27-05](https://user-images.githubusercontent.com/17137/93493670-2b39f900-f8da-11ea-8da0-0fd0276a3c4d.jpg)